### PR TITLE
added form

### PR DESCRIPTION
**Pull request by osisamkay**

### CODE REVIEW

**Comment by osisamkay** [comment]
Added Form

File: src/components/Home.jsx [hunk 1]
```diff
@@ -56,6 +56,7 @@ class Home extends Component {
         <Header>
           <h1>Recipouille</h1>
           <p>Your Trusted Source For Great Food Recipes</p>
+          <Form/>
```
Comment (josephrexme) [inline src/components/Home.jsx]: @osisamkay when using self-closing elements or components, leave a space before the slash. 
```suggestion
          <Form />
```
Make change and push back for merge

File: src/components/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+const Formstyle = styled.form`
+  margin-top:1.8em;
+ input{
```
Comment (josephrexme) [inline src/components/Form.jsx]: This input is nested in this form as well as the button so nest them one level deep

Comment (josephrexme) [reply src/components/Form.jsx]: The input should get a
```css
border: 0;
padding: 10px;
```
to make it look more like the design

File: src/components/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+`;
+
+
+const Form = () => (
+  <Formstyle>
+    <input type="text" name="recipe name" placeholder="Recipe name" />
+    <input type="text" name="ratings" placeholder="Ratings" />
+    <button type="submit">Create Recipie</button>
```
Comment (josephrexme) [inline src/components/Form.jsx]: Recipe is spelled wrong here
```suggestion
    <button type="submit">Create Recipe</button>
```
You can also add some padding to this button in the styles above

**Review by josephrexme — CHANGES_REQUESTED** [review]

File: src/components/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import styled from 'styled-components';
+
+
+const Formstyle = styled.form`
+  margin-top:1.8em;
+ input{
+   height:40px;
+   border-radius:var(--border-radius);
+   margin-right:20px;
+   width: 200px;
```
Comment (josephrexme) [inline src/components/Form.jsx]: This input should get
```suggestion
   width: 200px;
   border: 0;
   padding: 10px;
   font: inherit;
```
to make it look a little more like the design

**Pull request merged**